lib/ruby-push-notifications/mpns/mpns_notification.rb

```diff
@@ -0,0 +1,96 @@
+require 'builder'
+
+module RubyPushNotifications
+  module MPNS
+    # Encapsulates a MPNS Notification.
+    # Actually support for raw, toast, tiles notifications
+    # (http://msdn.microsoft.com/en-us/library/windowsphone/develop/hh202945)
+    #
+    class MPNSNotification
+
+      # @return [Array]. Array with the results from sending this notification.
+      attr_accessor :results
```

Comment:
ups.. may bad on documentation.. its return a MPNSResponse like GCM... doc from GCM is Array too..